dev.sh

```diff
@@ -0,0 +1,145 @@
+# A script to aid in the development of the project while integrating with Alexa Hosted Skills
+# Usage: ./dev.sh <command>
+
+# Exit on error
+set -e
+
+# Check if the command is provided
+if [ -z "$1" ]; then
+  echo "Usage: ./dev.sh <command>"
+  exit 1
+fi
+
+update_hosted_skill_repo(){
+    rsync -av --delete ../../../lambda ./
+    rsync -av --delete ../../../skill-package/interactionModels ./skill-package
+    rsync -av --delete ../../../skill-package/assets ./skill-package
+}
+
+resync_hosted_skill_repo(){
+    git checkout dev
+    git pull --rebase
+    git merge master
+    # If everything is managed via the script, then we can just push (no conflicts)
+    git push --no-verify
+    git checkout master
+}
+
+# Check if the command is valid, commands can be:
+# - init: Initialize the project with an existing skill id
+# - update: Update the skill hosted repo with the code from the local repo
+# - deploy: Update the model and Deploy the code from the skill hosted repo
+
+COMMAND=$1
+
+case $COMMAND in
+  new)
+
+    mkdir -p build
+    mkdir -p build/hosted
+    cd build/hosted
+
+    ask new
```

Comment:
should we add a check at the top of the file for the ASK cli?

Reply:
Done! ;)